elide-async/src/main/java/com/yahoo/elide/async/export/TableExporter.java

```diff
@@ -65,8 +65,8 @@ public TableExporter(Elide elide, String apiVersion, User user, GraphQLParser gr
                 projection = graphQLParser.parse(query);
                 //TODO - we need to add the baseUrlEndpoint to the queryObject.
                 //TODO - Can we have projectionInfo as null?
-                requestScope = new GraphQLRequestScope("", tx, user, apiVersion, elide.getElideSettings(), null,
-                        requestId);
+                requestScope = new GraphQLRequestScope("", tx, user, apiVersion, elide.getElideSettings(),
+                        null, requestId, null);
```

Comment:
If this is what is causing the NPE, instead of passing null, we can create an empty HashMap here.